### DISCUSSION TITLE
Can Bedrock embeddings be used for RAG ?

### DISCUSSION THREAD

**Discussion by thiagotps** [danny-avila/LibreChat#4714] [Help Wanted]
Looking at the  [documentation](https://www.librechat.ai/docs/configuration/rag_api#configuration), `EMBEDDINGS_PROVIDER` only accepts `openai`, `azure`, `huggingface`, `huggingfacetei`, or `ollama`as values. Is there some way to use the embedding models available through bedrock, like `amazon.titan-text-lite-v1` ?

**Comment by khfung** [comment]
I’m using it right now 
Here is all the info: https://github.com/danny-avila/rag_api

**Reply by AdamJJJ** [reply]
> I’m using it right now Here is all the info: https://github.com/danny-avila/rag_api

Could you provide more information how we put other variables ?
I'm putting this 
EMBEDDINGS_PROVIDER=bedrock
EMBEDDINGS_MODEL=cohere.embed-multilingual-v3

what should I put for these variable ?
RAG_API_URL=
RAG_PORT=8000

**Reply by caramale07** [reply]
Could you provide any updates? How did you configure that?
